### DISCUSSION TITLE
Licenses Screen

### DISCUSSION THREAD

**Discussion by Ryan365Apps** [keygen-sh/community#7] [General]
Hi Zeke,

Could we have filtering and sorting on the licenses table?
https://app.keygen.sh/licenses

Or do you anticipate users to create their own management tools using the API? 

I just want to be able to see expired, expiring soon , policy applied - that sort of thing

Thanks
R

**Comment by ezekg** [comment]
Currently, a lot of our customers build out their own portals, integrated into their other systems such as billing. But we are working on adding better UI filtering with our new upcoming dashboard redesign. We're hoping to have it out by the end of the year.

For now, you can use any API filters in the UI by manually updating the URL:

- Licenses expiring within next 3 days: https://app.keygen.sh/licenses?expiring=1
- Licenses that have expired: https://app.keygen.sh/licenses?expired=1
- Licenses for policy: https://app.keygen.sh/licenses?policy=POLICY_ID
- Licenses for user: https://app.keygen.sh/licenses?user=USER_ID

Other filters are here: https://keygen.sh/docs/api/#licenses-list
